content/blog/2022-05-28-gsoc2022-welcome.md

```diff
@@ -0,0 +1,42 @@
+---
+title: XMPP & Google Summer of Code 2022: Welcome new contributors!
+author: XSF Organization Administrator (GSoC 2022)
+date: 2022-05-28
+categories: ['Google Summer of Code']
+---
+
+![XSF and GSoC 2022 Logo](/images/logos/GSoC_2022_Logo.png)
+
+The Google Summer of Code 2022 is about to lift off and coding starts soon! The XSF has not just been 
+accepted (again!) as a hosting organization for XMPP projects, we also can welcome two new contributors who will work on open-source software projects in the XMPP environment! We have updated our [designated web-page](https://xmpp.org/community/gsoc-2022/) for the Google Summer of Code 2022 accordingly.
+
+### The XMPP projects at Google Summer of Code 2022
+
+So, please welcome **Patiga** and **PawBud** as new contributors! It is really great that you chose XMPP for your coding adventure!
+
+- **Patiga** will work on more [flexible file transfers in Dino](https://summerofcode.withgoogle.com/programs/2022/projects/z9ixHTWZ). Mentors will be **fiaxh** and **Marvin W.** - many thanks to both of you!
+  - Resource-wise, messenger applications tend to be on the lightweight side of the spectrum. This drastically changes when file transfers are added to the equation. File transfers can introduce arbitrary more resource-usage, both on network and data storage aspects. To alleviate this issue, stateless file sharing empowers the user to make informed decisions on which files to load. Deliverables:
+    - Unified handling of http and jingle (peer-to-peer) file transfers
```

Comment:
"of HTTP and Jingle (peer-to-peer) file transfers"

Reply:
Quality quality quality - thanks!

Reply:
4c8671fa6ab239881bd221f5f63006073242bb2a